chpl/chpl-resources/src/main/resources/system-triggers.xml

```diff
@@ -23,7 +23,8 @@
                 <job-name>downloadFileJob2014</job-name>
                 <job-group>systemJobs</job-group>
                 <misfire-instruction>MISFIRE_INSTRUCTION_DO_NOTHING</misfire-instruction>
-                <cron-expression>0 0 6 * * ?</cron-expression> <!-- At 0600 UTC every day -->
+                <cron-expression>0 45 10 ? * 1#1</cron-expression> <!-- At 1045 UTC on the first Sunday of the month -->
+                <!--<cron-expression>0 45 10 ? JAN,APR,JUL,OCT 1#1</cron-expression> At 1045 UTC on the first Saturday of the quarter -->
```

Comment:
Is this comment needed?

Reply:
That's going to be the schedule when it goes to quarterly in a month or two. Seemed reasonable to put it here for then, as it's a planned change, but I could remove it if necessary

Reply:
Ok, I'm fine with that then. Will resolve this one.